### PR TITLE
Remove attachments from case history exports

### PR DESCRIPTION
**Pull request by TylerSheffels**
Refer to: http://manage.dimagi.com/default.asp?160461#891283

This remove's attachments from case actions (which show up in the case history export)

### CODE REVIEW

File: couchexport/util.py [hunk 1]
```diff
@@ -45,6 +45,10 @@ def clear_attachments(schema_or_doc):
         del schema_or_doc['_attachments']
     if schema_or_doc and 'case_attachments' in schema_or_doc:
         del schema_or_doc['case_attachments']
+    if schema_or_doc:
```
Comment (dannyroberts) [inline couchexport/util.py]: Worth being over-protective/explicit here and adding `and schema_or_doc.get('doc_type') == 'CommCareCase'`?

Comment (czue) [inline couchexport/util.py]: yes in concept, though I think at this point `schema_or_doc` is a CommCareCaseAction instance: https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/models.py#L57

Maybe we can do something sketchy like checking for `updated_known_properties`/`updated_unknown_properties` or similar?

Definitely want to limit the likelihood of someone naming a question `"attachments"` and not being able to export it.

Comment (TylerSheffels) [inline couchexport/util.py]: Makes sense, after taking a look I don't see anything notably less sketchy to check for. I'll update.

Comment (TylerSheffels) [inline couchexport/util.py]: @czue @dannyroberts I updated below, but after updating I don't think a question named attachments would get removed.  That questions would either be under `updated_known_properties` or `updated_unknown_properties`, correct?

Comment (czue) [inline couchexport/util.py]: yeah, unless a form had all three of those properties, which would be pretty surprising/rare.

**Pull request merged**